src/components/dropdown/Dropdown.tsx

```diff
@@ -326,6 +330,7 @@ export class Dropdown<T = LabelValue> extends React.Component<
           isOptionSelected={this.handleIsOptionSelected}
           menuPlacement="auto"
           maxMenuHeight={maxMenuHeight}
+          mode={mode ? mode : 'aligned'}
```

Comment:
Why adding the class without any specific css?

Reply:
It's used here actually
https://github.com/SymphonyPlatformSolutions/symphony-bdk-ui-toolkit-components/pull/207/files#diff-19565f42e196c6bb4854c8d839424797c071cb733757393a6d72f1b400ce2f9eR55

I added this line (`mode={mode ? mode : 'aligned'}`) because it was for me the only way to make it available on the custom render (through props?.selectProps?.mode)

Unless there is another way?